charts/flyte-core/templates/admin/service-monitor.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if and .Values.flyteadmin.serviceMonitor.enabled .Values.flyteadmin.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "flyteadmin.name" . }}
+  namespace: {{ template "flyte.namespace" . }}
+  labels:
+  {{- with .Values.flyteadmin.serviceMonitor.labels }}
+  {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  endpoints:
+    - interval: 60s
```

Comment:
NIT: Do we want to expose the interval and scrapeTimeout as values that can be set under Values.flyteadmin.serviceMonitor?

Reply:
Added.